xc7/utils/prjxray_routing_import.py

```diff
@@ -52,6 +52,14 @@
 HCLK_OUT = re.compile('CLK_HROW_CK_HCLK_OUT_([LR])([0-9]+)')
 IOI_OCLK = re.compile('IOI_OCLK_([01])')
 
+# Regex for [LR]IOI_SING tiles
+IOI_SITES = ['OLOGIC', 'ILOGIC', 'IDELAY', 'OCLK_', 'OCLKM_']
```

Comment:
I'd also rename that variable. `OCLK_` and `OCLKM_`  are not sites.

Reply:
Done